app/models/feedback_session.rb

```diff
@@ -0,0 +1,21 @@
+class FeedbackSession
+  class FeedbackVariableRequiredError < StandardError; end
+  class WrongDateFormatError < StandardError; end
+  class InvalidDateError < StandardError; end
+
+  def self.missing(from_str_date, to_str_date)
+    raise FeedbackVariableRequiredError unless Variable.feedback
```

Comment:
@radinreth better check if Variable.feedback.nil? rather than to keep ourselves think reversible

Reply:
Note  bong